projects/qtaf/docs/sections/plugins/xray/Configuration.md

```diff
@@ -0,0 +1,1383 @@
+      <td>
+        <ol>
+          <li style="color:green">Click register</li>
+          <li style="color:green">Enter username</li>
+          <li style="color:green">Enter password</li>
+          <li style="color:green">Enter address</li>
+          <li style="color:green">Click submit</li>
+        </ol>
+      </td>
+      <td>
+        <ol>
+          <li style="color:green">Click register</li>
+          <li style="color:green">Enter username</li>
+          <li style="color:green">Enter password</li>
+          <li style="color:green">Enter address</li>
+          <li style="color:gray">Click submit</li>
+        </ol>
+      </td>
+      <td>Mismatched steps in interation 2: the address was never actually entered and Xray marked the fifth step as TODO because only four steps were actually executed in iteration 2.</td>
+    </tr>
+  </tbody>
+</table>
+
+Merging the iterations' steps can therefore help for data-driven testing.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_TESTS_STEPS_MERGE`
+
+***Type***
+: `boolean`
+: Accepted strings (case-insensitive):
+
+    - `#!json "true"`
+      `#!json "1"`
+      `#!json "y"`
+
+    - `#!json "false"`
+      `#!json "0"`
+      `#!json "n"`
+
+***Default***
+: `#!json false`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "tests": {
+              "info": {
+                "steps": {
+                  "merge": true
+                }
+              }
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_TESTS_STEPS_MERGE="true"
+        ```
+
+<hr/>
+
+## `service`
+
+Defines whether the targeted Xray instance is an Xray cloud or Xray server instance.
+QTAF requires this setting because Xray's APIs need to be addressed slightly differently.
+
+***Environment variable***
+: `XRAY_SERVICE`
+
+***Type***
+: `#!json "cloud"` or `#!json "server"`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+	        "service": "server"
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_SERVICE="server"
+        ```
+
+<hr/>
+
+## `url`
+
+To properly connect to Xray, QTAF needs to know where the Xray and Jira instances can be found.
+
+<hr/>
+
+### `jira`
+
+Defines the base URL of the Jira instance.
+For Jira cloud, it is usually of the form `https://your-domain.atlassian.net` (without the `/jira` part, see [here](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/#ad-hoc-api-calls/)).
+For Jira server, you can have a look [here](https://confluence.atlassian.com/adminjiraserver/configuring-the-base-url-938847830.html) to determine your base URL.
+
+***Environment variable***
+: `XRAY_URL_JIRA`
+
+***Type***
+: `string`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+	        "url": {
+            "jira": "https://example.org/development/jira"
```

Comment:
The indentation does not look good

Reply:
Fixed in bdf5080ab870aac828d9e70d10cab3594ba08cd2.